scripts/grok1_multiblock_lib.py

```
@@ -453,13 +616,30 @@ def _load_or_build(self, name: str) -> tuple[np.ndarray, np.ndarray]:
         ref = np.asarray(self._ref.vector(role), dtype=np.float32)
         ref_shape = tuple(int(d) for d in ref.shape)
         q_path, s_path = self._paths(name)
-        if q_path.is_file() and s_path.is_file():
+
+        # Load the bit-identical INT4 codes. They live in the absmax table
+        # directory so absmax and LS-α modes never duplicate them.
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `α` (GREEK SMALL LETTER ALPHA). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF003)